trunk/NDHMS/Routing/module_HYDRO_io.F

```diff
@@ -2943,30 +2931,30 @@ subroutine sub_output_gw(igrid, split_output_count, ixrt, jxrt, nsoil, &
         xdumd = qsgwrt
 #endif
 
-     iret = nf_inq_varid(ncid,  "QSGWRT", varid)
-     iret = nf_put_vara_real(ncid, varid, (/1,1,output_count/), (/ixrtd,jxrtd,1/), xdumd)
+     iret = nf90_inq_varid(ncid,  "QSGWRT", varid)
+     iret = nf90_put_var(ncid, varid, xdumd, (/1,1,output_count/), (/ixrtd,jxrtd,1/))
      
 #ifdef MPP_LAND
         xdumd = gQgw_chanrt
 #else  
         xdumd = qgw_chanrt
 #endif
 
-     iret = nf_inq_varid(ncid,  "QGW_CHANRT", varid)
-     iret = nf_put_vara_real(ncid, varid, (/1,1,output_count/), (/ixrtd,jxrtd,1/), xdumd)
```

Comment:
This pattern is not trivial to replace and happens frequently. Just seems worth noting.

Reply:
What do you mean not trivial to replace?